lhotse/bin/modes/recipes/gigast.py

```diff
@@ -0,0 +1,68 @@
+import logging
+from typing import List
+
+import click
+
+from lhotse.bin.modes import download, prepare
+from lhotse.recipes.gigast import GIGASPEECH_PARTS, download_gigast, prepare_gigast
+from lhotse.utils import Pathlike
+
+
+@prepare.command(context_settings=dict(show_default=True))
+@click.argument("corpus_dir", type=click.Path(exists=True, dir_okay=True))
+@click.argument("manifests_dir", type=click.Path(exists=True, dir_okay=True))
+@click.argument("output_dir", type=click.Path())
+@click.option(
+    "-l",
+    "--language",
+    default=["auto"],
```

Comment:
You can add `type=click.Choice(("auto",) + GIGAST_LANGS)` here so that the CLI help is more informative about which languages are supported.

Reply:
OK, I've added this.